### ISSUE TITLE
minimum identity and MUMmer

### ISSUE THREAD

**Issue by kneubehl** [moshi4/pyGenomeViz#8]
Hi,

Sorry for another question. I am using the pgv-mummer with the --seqtype specified as nucleotide and I can't seem to get any sequence identities to plot lower than 80% even when specifying the --min_identity. Any guidance would be appreciated. Thanks!

**Comment by moshi4** [comment]
Are you saying that even though there are sequence similarity results for identities below 80% in the `align_coords.tsv` output of pgv-mummer, they cannot be visualized?

Also, there may be a misunderstanding regarding the `--min_identity` option setting, so I'll explain just in case. As an example, the option setting `--min_identity=80` means to visualize 80-100% identity link results.

**Comment by kneubehl** [comment]
Sorry this was my bad. I looked into MUMmer a little more and it works best for similar genomes. I am looking at different genomic regions that have <80% nucleotide identity and I don't think MUMmer looks at those. I tried the minimum identity option to see if I can get it to show something if I changed it but no luck. In the align_coords.tsv there is nothing below 80%. I think my best option is the protein sequence option.
